CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 * Requesting the alignment without also requesting the sequence for BAM files containing empty CIGAR strings does now
   not result in erroneous parsing ([\#2418](https://github.com/seqan/seqan3/pull/2418)).
+* BAM files with 64 references are now parsed correctly ([\#2423](https://github.com/seqan/seqan3/pull/2423)).
```

Comment:
Technically it's every number of reference whose last byte is `\x40`, e.g. it's also affecting `320`, but I didn't come up with a concise way to describe this.

Reply:
(mod 64) ?

Reply:
Well, it works properly for 128. The first byte has to look like `01000000` or `11000000`, i.e. the first least significant bit to be set is the seventh...

Reply:
you are right: (mod 256) == 64